### DISCUSSION TITLE
Print migrated rules

### DISCUSSION THREAD

**Discussion by kurtextrem** [biomejs/biome#4332] [Ideas]
Hey, so the migration tool is great for _switching_ from eslint to biome, but it's not great if you plan to use both. I'm currently manually comparing which rules are turned on and which rules Biome provides, so I can turn off those rules in eslint.
It'd be awesome to have the migration rule print out something like
> Put this in your eslint config:
> "rule/foo": "off",
> "rule/bar": "off"

so you can easily turn off everything that has been migrated.

**Comment by Conaclos** [comment]
Thsi is related to #3282.
I opened a [task](https://github.com/biomejs/biome/issues/3397) to create a migration report.
